packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -1026,57 +1086,7 @@ export default function RouteBuilder({
       form={form}
       appUrl={appUrl}
       enrichedWithUserProfileForm={enrichedWithUserProfileForm}
-      Page={function Page({ hookForm, form }) {
```

Comment:
We were redefining the component causing React state to get reset.

This wasn't causing issue earlier as there was no re-render in SingleForm earlier it seems but now there is.